backend/routes/projectById.js

```diff
@@ -0,0 +1,34 @@
+const { parse } = require("url");
+
+const { missingProperty, invalidProperty } = require('../src/utils/errors');
+const stringify = require('../src/utils/stringify');
+const getAllProjects = require('../src/components/projects/getAllProjects');
+
+module.exports = async (req, res) => {
+  const { query: { project: projectParam } } = parse(req.url, true);
+
+  if (!projectParam) {
+    res.statusCode = 422;
+    res.end(stringify({
+      code: 422,
```

Comment:
Why put the status code in two places? If it's really necessary maybe it should reference one variable -> single source of truth.

Reply:
Actually it is kinda according to `json api` specifications. But assigning it to the variable is a good point 👍